spec/features/user_registers_spec.rb

```diff
@@ -0,0 +1,73 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.